src/views/Favorites.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+
+import Utils from '../utils/Utils';
+
+import favoritesActions from '../actions/favorites';
+import ListGifs from '../components/ListGifs';
+import SearchField from '../components/SearchField';
+
+
+class ViewFavorites extends Component {
+
+    componentWillMount() {
+        this.props.getFavoritesList();
+        this.callAction = Utils.debounce(this.callAction, 500);
```

Comment:
`this.callAction` belongs to constructor

Reply:
yeah! I search that this past week (calls in componentDidMount vs WillMount) and you are right